src/SamplesApp/UITests.Shared/Windows_Media/MediaPlayerTests.xaml.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using Uno.UI.Samples.Controls;
+using Uno.UI.Samples.UITests.Helpers;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace UITests.Windows_Media
+{
+	[SampleControlInfo("Windows.Media", "MediaPlayer", viewModelType: typeof(MediaPlayerTestsViewModel))]
+	public sealed partial class MediaPlayerTests : Page
+	{
+		public string SoundToPlay = "ms-appx:///sound.mp3";
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6277035)